appveyor.yml

```diff
@@ -0,0 +1,16 @@
+environment:
+  nodejs_version: "6"
```

Comment:
I think Travis gives us good coverage of multiple node versions, though there might be some environmental issues that only affect certain versions of node.